Admin order show#37

timnallen opened the pull request:
collaborator
As an admin user
When I visit a user's profile
And I click on a link for order's show page
My URL route is now something like "/admin/orders/15"
I see all information about the order, including the following information:
- the ID of the order
- the date the order was made
- the date the order was last updated
- the current status of the order
- each item the user ordered, including name, description, thumbnail, quantity, price and subtotal
- the total quantity of items in the whole order
- the grand total of all items for that order

closes #37 

Had to change some routes from my earlier PRs to make this work... This is clearly closer to what they were looking for. This could easily be DRYed up later on with the use of partials.

timnallen committed:
merges merchant dashboard into my branch

timnallen committed:
Cleaned up some organization in factory files, and added a missing end from a merge conflict oversight in the order model

timnallen committed:
adds setup to admin order show spec, added user_id as params in profile_orders_path so admin can access the orders for a user who isn't logged in also reflected in OrdersController

timnallen committed:
Reverted back to old OrdersController, added link to namespaced admin route for orders index and route itself.

timnallen committed:
Adds admin orders controller and index action

timnallen committed:
Adds view for admin::orders#index

timnallen committed:
adds orders info to admin orders index

timnallen committed:
Adds orders as instance variable in Admin::OrdersController

timnallen committed:
adds show action to Admin::ORdersController

timnallen committed:
Adds order as instance variable and all info on view for admin orders show

timnallen committed:
Adds spec for all contents on admin order show page

allieIsOkay reviewed (approved):
Looks good, thanks Tim.

timnallen committed:
Update items_controller.rb

Based on Peter's observation, I'm making this simpler.

timnallen committed:
Update item.rb

I'm removing this from the PR, and it may cause a merge conflict, if we don't remove it before the merge, but we can debate its uses then.

timnallen committed:
Update routes.rb

Same deal, here... In case we do remove these, I don't want my PR to cause the merge conflict.

timnallen committed:
Update item_spec.rb

Removing the spec for this, as well.

teresa-m-knowles committed:
Merge branch 'master' into admin_order_show#37

teresa-m-knowles reviewed (approved):
Looks good, thanks Tim!

csvlewis commented:
Good work, I'll merge it!
